data/ui/piper.ui

```diff
@@ -1,927 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!-- Generated with glade 3.19.0 -->
```

Comment:
I think it's best if you extract that commit to master, git push it and then rebase the translation branch on top. otherwise this is mixed in with the translation bits, which is a bit odd.